rest-api/cron.yaml

```diff
@@ -1,7 +1,8 @@
 cron:
 - description: Daily metrics
   url: /offline/MetricsRecalculate
-  schedule: every day 01:00
+  schedule: every day 04:00
```

Comment:
Do you mean for this to be in the same PR? Would you mind either calling it out in the PR title or splitting it up? I'd like it to show up in release notes.

Reply:
Added to PR title.